models/gc_grt123_lung_cancer/meta.json

```diff
@@ -0,0 +1,171 @@
+{
+  "id": "2e67a3cc-4680-4058-bf4e-f965cf50f06f",
+  "name": "lung_cancer_risk_estimation",
+  "title": "Lung cancer risk estimation on thorax CT scans",
+  "summary": {
+    "description": "This algorithm analyzes non-contrast CT scans of the thorax and predicts the lung cancer risk. The model consists of two modules. The first one is a 3D region proposal network for nodule detection, which outputs all suspicious nodules for a subject. The second one selects the top five nodules based on the detection confidence, evaluates their cancer probabilities and combines them with a leaky noisy-or gate to obtain the probability of lung cancer for the subject. This model was the winner of the Data Science Bowl 2017 competition hosted on Kaggle.",
+    "inputs": [
+      {
+        "label": "CT",
+        "description": "Chest CT",
+        "format": "DICOM",
+        "modality": "CT",
+        "bodypartexamined": "Chest",
+        "slicethickness": "2.5mm",
+        "non-contrast": true,
+        "contrast": false
+      }
+    ],
+    "outputs": [
+      {
+        "type": "Prediction",
+        "valueType": "number",
+        "label": "Cancer probability score",
+        "description": "Probability that the scan contains cancer nodules",
```

Comment:
It's not the entire scan but just the lungs, right?

Reply:
Correct, probability that there are any cancer nodules in the lungs. I'll make it more specific.